templates/email/messages/order_cancelled_en.html

```diff
@@ -0,0 +1,22 @@
+<p>Dear customer, we have received your cancellation. The invoice, application and agreement related to the berth are cancelled as of {{ order.rejectedAt }}.</p>
```

Comment:
The prop is on `snake_case`, same for the other templates. :)
```suggestion
<p>Dear customer, we have received your cancellation. The invoice, application and agreement related to the berth are cancelled as of {{ order.rejected_at }}.</p>
```

Reply:
Fixed